libs/src/sdk/api/playlists/PlaylistsApi.ts

```diff
@@ -518,4 +576,42 @@ export class PlaylistsApi extends GeneratedPlaylistsApi {
     }
     return trackMetadata
   }
+
+  /**
+   * Update helper method that first fetches a playlist and then updates it
+   */
+  private async fetchAndUpdatePlaylist(
```

Comment:
🔥

Reply:
should we use this for `updatePlaylist` too? so that the consumer can just pass in the changed fields

Reply:
I was planning on leaving `updatePlaylist` as is because in many cases the consumer will have fetched the playlist anyway before updating it. In the case for reordering tracks for example, the consumer will need to have the original order fetched and then can reorder

But I would like to have a bigger discussion around entity manager supporting partial updates